### PR TITLE
Invitation PubSub

### LINKED ISSUE #434: Invitation PubSub

### PR DESCRIPTION
**Pull request by amos-kibet**
Resolves #434.

### CODE REVIEW

File: lib/omedis_web/live/invitation_live/index.ex [hunk 1]
```diff
@@ -3,22 +3,42 @@ defmodule OmedisWeb.InvitationLive.Index do
...
      |> stream(:invitations, [])}
   end
 
+  defp maybe_subscribe_to_invitation_updates(current_organisation) do
+    case current_organisation do
+      nil ->
+        :ok
+
+      %Organisation{id: current_organisation_id} ->
+        :ok = Endpoint.subscribe("invitation:accepted:#{current_organisation_id}")
```
[the remark this answers is not in the document]
Comment (almirsarajcic) [reply lib/omedis_web/live/invitation_live/index.ex:35]: If it's not possible to prefix with the `organisation_id`, then put it after `invitation`. Is there a suffix option? In that case `invitation` would be suffix.

[the remark this answers is not in the document]
Comment (amos-kibet) [reply lib/omedis_web/live/invitation_live/index.ex:35]: I'll investigate the suffix option. If it does not exist, we can skip the prefix, then generate topics this way:
```elixir
ublish :accept, [:organisation_id, "invitation"]
  publish :create, [:organisation_id, "invitation"]
  publish :destroy, [:organisation_id, "invitation"]
  publish :expire, [:organisation_id, "invitation"]
```

that should be possible. Let me try the first option first, then maybe this one.

[the remark this answers is not in the document]
Comment (almirsarajcic) [reply lib/omedis_web/live/invitation_live/index.ex:35]: 🤞

[the remark this answers is not in the document]
Comment (amos-kibet) [reply lib/omedis_web/live/invitation_live/index.ex]: I went with `organisation_id:invitations` topic format.

**Review by almirsarajcic — APPROVED** [review]

**Pull request merged**